core/src/main/java/io/questdb/std/MemoryTag.java

```diff
@@ -48,6 +48,7 @@ public final class MemoryTag {
     public static final int NATIVE_CB5 = 32;
     public static final int NATIVE_COMPACT_MAP = 8;
     public static final int NATIVE_DEFAULT = 1;
+    public static final int NATIVE_DIRECT_BYTE_SINK = 54;
```

Comment:
This change requires additional stuff done here: add an entry to `tagNameMap` and update `SIZE`'s value.

Reply:
Yup, just noticed a failing test case :-) Fixing.